scripts/config_hdf5.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+#########################################################################
+# 
+#                 #######               ######  #######
+#                 ##                    ##   ## ##
+#                 #####   ##  ## #####  ##   ## ## ####
+#                 ##       ####  ## ##  ##   ## ##   ##
+#                 ####### ##  ## ###### ######  #######
+#
+#  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+#
+#  Copyright (C) 2021 by the ExaDG authors
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+#########################################################################
+
+
+./configure --prefix=$WORKING_DIRECTORY/hdf5/install/ --enable-unsupported --enable-symbols=yes --disable-threadsafe --disable-cxx --enable-hl --without-szlib --enable-parallel --enable-shared CC=mpicc
```

Comment:
I think we should use separate rows for every option for readability, agree? See for comparison e.g. the configure script for PETSc.

Reply:
I agree, I just copied the line from a script I had.